vendor/bat-native-ledger/src/bat_helper.cc

```diff
@@ -607,7 +607,7 @@ static bool ignore_ = false;
     allow_videos_ = state.allow_videos_;
     monthly_balances_ = state.monthly_balances_;
     recurring_donation_ = state.recurring_donation_;
-    migrate_score = state.migrate_score;
+    migrate_score_2 = state.migrate_score_2;
```

Comment:
score_2?

Reply:
so we migrated once already and we just need to migrate it again, so that's why I rename it to 2 as 1 is not needed anymore

Reply:
but why does it need to be score_2 in the data structure?

Reply:
so that we migrate only once

Reply:
I see, the name has changed now, but I think what we need here is a `version` attribute for the data structure